user/build-lifecycle.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Customizing the Build
+layout: en
+---
+
+Travis CI provides a default build environment and a default set of phases for each programming language. You can customize any phase in this process in your `.travis.yml`.
+
+<div id="toc"></div>
+
+## The Build Lifecycle
+
+A build or job on Travis CI is made up of two main parts:
```

Comment:
Should we use "job" here, instead of "build or job"? I think it might help clarify the concept that, at the end, what is running these commands is a job.

I might be missing background on some previous decisions/discussions/conventions, so please excuse me if this is on purpose :) I think this PR might be also about "moving things around" so apologies if this is a too detailed comment!

Reply:
This is great, and a valid point!